### ISSUE TITLE
DOI publication lookup leaves out latin binomials?

### ISSUE THREAD

**Issue by cjconroy** [ArctosDB/arctos#2763]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Describe the bug**
A couple of us (CJC and CLS) at MVZ have noticed randomly that pubs gotten through the DOI lookup lack the latin words. 

**To Reproduce**
Pull a publication with a DOI. Unfortunately, I rechecked the one I saw yesterday, and it works fine. Has anyone else seen this behavior?

The pub I saw yesterday looked like this. It is 2019, so it was loaded fairly recently. 
Silvia E Pavan. 2019. A revision of the group (Didelphidae: Marmosini), with a description of a new species from Roraima, Brazil. Journal of Mammalogy 100(1):103-117.
But should have been 
Silvia E Pavan. 2019. A revision of the Monodelphis glirina group (Didelphidae: Marmosini), with a description of a new species from Roraima, Brazil. Journal of Mammalogy 100(1):103-117.

**Expected behavior**
That all words in a publication come along. 

**Desktop (please complete the following information):**
 - Mac
 - Chrome
 - Latest version

**Additional context**
This is supposed to be a tool to save time, but if we have to lookup every DOI and title word by word, then it is not. 

**Priority**
Medium

**Comment by dustymc** [comment]
I rebuilt the function to use crossref's JSON services, which are much nicer. Crossref also has a formatter option now, which is amazing. I've just got a couple options hard-coded in a dropdown on create publication, at some point we might pull some or all of http://api.crossref.org/styles into a code table. (Probably "some" - surely nobody wants to scroll through 921 cryptic values?)

<img width="776" alt="Screen Shot 2020-06-11 at 3 56 47 PM" src="https://user-images.githubusercontent.com/5720791/84447179-33f4a200-abfc-11ea-8d43-e2b465091d9b.png">

Let me know if you want something else from the list and I'll add it.

The formatted publication for 10.3099/MCZ42.1 is still a bit of a mess, I'm stripping linebreaks and returns now but it's coming from Crossref in every format as ``...Boundaries In theHydromantes shastaeComplex,...``, which I can't do much about.

Basic functionality should be in the next release, suggest we add a 'refresh as {format}' option to edit publication for dealing with old and funky full citations.
